src/components/HomeSearchBox/index.tsx

```diff
@@ -1,42 +1,110 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import { useRouter } from 'next/router'
 import useTranslation from 'next-translate/useTranslation'
 
 const HomeSearchBox = () => {
+  const [searchValue, setSearchValue] = useState<string>('')
+  const [active, setActive] = useState(0)
+  const [filteredSuggestion, setFilteredSuggestion] = useState<string[]>([])
+  const [isSuggestionShown, setSuggestionShown] = useState(false)
   const { t } = useTranslation('home')
   const router = useRouter()
-  const [searchValue, setSearchValue] = useState<string>('')
+
+  useEffect(() => {
+    const handleMouseClick = (event: MouseEvent) => {
+      setSuggestionShown(false)
```

Comment:
Now when clicking suggestion item, it can redirect to another page.

```suggestion
      const isFocused: boolean | undefined = searchBoxRef.current?.contains(event.target as Node)

      if (!isFocused) {
        setSuggestionShown(false)
      }
```